src/bors/command/parser.rs

```diff
@@ -70,6 +70,27 @@ impl<'a> Tokenizer<'a> {
 
 type ParseResult<'a> = Option<Result<BorsCommand, CommandParseError<'a>>>;
 
+
+//  parse a line of text into command and key-value pairs
+fn parse_key_value(line: &str) -> Option<(&str, Vec<(&str, &str)>)> {
```

Comment:
I think that it would make sense for this function to return the key-value pairs in a `HashMap<String, String>` (or `HashMap<&str, &str>`, doesn't really matter), the order shouldn't be important (at least we don't have a use-case for it being important now).